libs/num/uint.go

```diff
@@ -111,6 +111,15 @@ func UintFromDecimal(d Decimal) (*Uint, bool) {
 	return &Uint{*u}, ok
 }
 
+// UintFromUint64 allows for the conversion from uint64.
+func UintFromUint64(ui uint64) *Uint {
```

Comment:
How does it compare with NewUint()?

Reply:
Removed it and used `NewUint()` instead. Didn't see it originally because I was looking for a `UintFromBlah()` named function.